registration/registration-services/src/main/java/io/mosip/registration/constants/RegistrationConstants.java

```diff
@@ -1493,6 +1493,9 @@ public static List<String> fieldsToExclude() {
 	public static final String PUBLIC_KEY_REF_ID = "packet-encryption-key";
 	public static final String USER_DETAIL_SALT_SERVICE_NAME = "user_salt_details";
 	public static final String SERVICES_VERSION_KEY = "mosip.reg.services.current.version";
+	public static final String VERSION_MAPPINGS_KEY = "mosip.registration.verion.upgrade.version-mappings";
+	public static final String ROLLBACK_FILE_PATH = "mosip.reg.rollback.path";
+	public static final String MANIFEST_PATH = "/MANIFEST.MF";
```

Comment:
lets not hardcode the "/", instead when creating the full path user File.separator, Also cross-check how are we taking "mosip.reg.rollback.path"